packages/client/typesafe-css-modules.ts

```diff
@@ -8,7 +8,7 @@ import * as prettier from 'prettier';
 import prettierConfig from '../../.prettierrc.json';
 
 // Directoryes to write file to
-const targets = ['./', '../server/', '../shared/'];
+const targets = ['./', '../server/'];
```

Comment:
Var en del krøll med declaration filene, fjernet shared som et target for nå siden den sjekkingen ikke er driftskritisk. 

PS: Vi må flytte css modul filene i shared til klienten.